Sources/Interaction/Style/InteractorStyleManipulator/Presets.js

```diff
@@ -14,6 +14,7 @@ const MANIPULTOR_TYPES = {
   gestureCamera: Manipulators.vtkGestureCameraManipulator,
   movement: Manipulators.vtkKeyboardCameraManipulator,
   freeLook: Manipulators.vtkMouseCameraTrackballFirstPersonManipulator,
+  unicam: Manipulators.vtkMouseCameraUnicamManipulator,
```

Comment:
Should the rotate manipulator be added here ?

Reply:
I agree